verticapy/tests/vModel/test_decision_tree_classifier.py

```diff
@@ -308,10 +308,8 @@ def test_get_params(self, model):
         params = model.get_params()
 
         assert params == {
-            "n_estimators": 1,
```

Comment:
i am not sure why you are deleting these parameters. please double check. if you have taken these parameters out then its alright.